src/brijesh/LoopAssignment8.java

```diff
@@ -0,0 +1,56 @@
+package brijesh;
+
+public class LoopAssignment8 {
+
+	void displayEvenNumber(int start, int end) {
+		System.out.println("Even numbers are:");
+		int index= start;
```

Comment:
You can handle this scenario without creating third variable, think on this.
can you use start++ for increment?

Reply:
Yes you are right. I can directly write something like this so we can eliminate one extra variable. Thanks for pointing out.
```
while(start<=end) {
			if(index%5==0)
				System.out.println(start);
			start++;
		}
```